pubspec.lock

```diff
@@ -36,6 +36,13 @@ packages:
       url: "https://pub.dartlang.org"
     source: hosted
     version: "2.5.0"
+  audio_session:
```

Comment:
Avoid committing this file until you clean it on your repository

Reply:
add this file to .gitignore it gets generated in each build anyway

Reply:
already in gitignore, but git still keeps pushing it

Reply:
this is probably because you added it later. check this [link](https://stackoverflow.com/questions/18418718/git-still-adds-and-tracks-folders-marked-in-gitignore)

also if you are using android studio, search the plugins market for .gitignore plugin, it will handle everything regarding the .gitignore file for you, this is for future projects ^^